src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -1060,7 +1060,8 @@ protected void ValidateComputerName(string[] computerNames)
         /// <returns>Parameter value as string.</returns>
         private static string GetSSHConnectionStringParameter(object param)
         {
-            if (param is string paramValue && !string.IsNullOrEmpty(paramValue))
+            string paramValue = LanguagePrimitives.ConvertTo<string>(param);
```

Comment:
ConvertTo() can throw PSInvalidCastException but we should keep PSArgumentException.

Reply:
Is there a circumstance where converting to `string` will throw?

Reply:
Yes, it is not common but possible (if someone creates ToString() method which throw).